File: Calculadora.cs
using System;


class Calculadora
{
    static void Main()
    {
        Console.WriteLine("Ingrese el primer número:");
        double num1 = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Ingrese el segundo número:");
        double num2 = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("Elija una operación (+, -, *, /):");
        char operacion = Console.ReadLine()[0];

        double resultado;

        switch (operacion)
        {
            case '+':
                resultado = num1 + num2;
                Console.WriteLine("Resultado: " + resultado);
                break;
            case '-':
                resultado = num1 - num2;
                Console.WriteLine("Resultado: " + resultado);
                break;
            case '*':
                resultado = num1 * num2;
                Console.WriteLine("Resultado: " + resultado);
                break;
            case '/':
                if (num2 != 0)
                {
                    resultado = num1 / num2;
                    Console.WriteLine("Resultado: " + resultado);
                }
                else
                {
                    Console.WriteLine("Error: No se puede dividir entre cero.");
                }
                break;
            default:
                Console.WriteLine("Operación no válida.");
                break;
        }
    }
}

File: calculadora.js
// Función principal de la calculadora
function calculadora() {
    // Pedir el primer número al usuario
    const num1 = parseFloat(prompt("Ingrese el primer número:"));
    
    // Pedir el segundo número al usuario
    const num2 = parseFloat(prompt("Ingrese el segundo número:"));
    
    // Pedir la operación al usuario
    const operacion = prompt("Elija una operación (+, -, *, /):");
    
    let resultado;
    
    // Realizar la operación seleccionada
    switch (operacion) {
        case "+":
            resultado = num1 + num2;
            break;
        case "-":
            resultado = num1 - num2;
            break;
        case "*":
            resultado = num1 * num2;
            break;
        case "/":
            if (num2 !== 0) {
                resultado = num1 / num2;
            } else {
                resultado = "Error: No se puede dividir entre cero";
            }
            break;
        default:
            resultado = "Operación no válida";
            break;
    }
    
    // Mostrar el resultado en una alerta
    alert("Resultado: " + resultado);
}

// Llamar a la función calculadora
calculadora();